app/scripts/lib/setupSentry.js

```diff
@@ -666,9 +666,16 @@ export default function setupSentry({ release, getState }) {
   const startSession = async () => {
     const hub = Sentry.getCurrentHub?.();
     const options = hub.getClient?.().getOptions?.() ?? {};
-    if (hub && (await getSentryEnabled()) === true) {
+    const isMetaMetricsEnabled = (await getMetaMetricsEnabled()) === true;
+
+    if (hub && isMetaMetricsEnabled) {
       options.autoSessionTracking = true;
-      hub.startSession();
+
+      try {
+        Sentry.getCurrentHub().startSession();
+      } finally {
+        Sentry.getCurrentHub().captureSession();
```

Comment:
I think ideally the durations and updates for the scopes are handled by Sentry internally, and align with page loads for example in the UI, though this `startSession` here provides us with a background session also.

As we now hopefully have a full fix for the session count issue in a subsequent PR, can we omit this `captureSession` as I don't believe it's necessary.

Reply:
yes definitely, this makes sense, adjusted in [1b9a62](https://github.com/MetaMask/metamask-extension/pull/25990/commits/1b9a62b2bc3cee88dbe709a49739805082a11fc8)